examples/workflow/workflow.py

```diff
@@ -0,0 +1,80 @@
+    workflowComponent = "dapr"
+    workflowName = "PlaceOrder"
+    workflowOptions = dict()
+    workflowOptions["task_queue"] =  "testQueue"
+    inventoryItem = ("Computers", 5, 10)
+    item2 = "paperclips"
+    eventName = "ChangePurchaseItem"
+    eventData = "eventData"
+    nonExistentIDError = "No such instance exists"
+    # Wait for sidecar to be up within 5 seconds.
+    d.wait(5)
+
+    sleep(5)
+
+    # Start the workflow
+    start_resp = d.start_workflow(instance_id=instanceId, workflow_component=workflowComponent,
+                     workflow_name=workflowName, input=item2, workflow_options=workflowOptions)
+    print(f"Attempting to start {workflowName}")
+    print(f"start_resp {start_resp.instance_id}")
+    getResponse = d.get_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    print(f"Get response from {workflowName} after start call: {getResponse.runtime_status}")
+
+    # Pause Test
+    d.pause_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    getResponse = d.get_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    print(f"Get response from {workflowName} after pause call: {getResponse.runtime_status}")
+
+    # Resume Test
+    d.resume_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    getResponse = d.get_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    print(f"Get response from {workflowName} after resume call: {getResponse.runtime_status}")
+
+    # Terminate Test
+    d.terminate_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    getResponse = d.get_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    print(f"Get response from {workflowName} after terminate call: {getResponse.runtime_status}")
+
+    # Purge Test
+    d.purge_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    try:
+        getResponse = d.get_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    except DaprInternalError as err:
+        if nonExistentIDError in err._message:
+            print("Instance Successfully Purged")
+
+    # Second test in order to test raise event #
+
+    # Start the workflow
+    start_resp = d.start_workflow(instance_id=instanceId, workflow_component=workflowComponent,
+                     workflow_name=workflowName, input=item2, workflow_options=workflowOptions)
+    print(f"Attempting to start {workflowName}")
+    print(f"start_resp {start_resp.instance_id}")
+    getResponse = d.get_workflow(instance_id=instanceId, workflow_component=workflowComponent)
+    print(f"Get response from {workflowName} after start call: {getResponse.runtime_status}")
+
+    # Raise event
+    d.raise_workflow_event(instance_id=instanceId, workflow_component=workflowComponent,
+                  event_name=eventName, event_data=eventData)
+    
+    # Sleep so that the workflow can finish 
+    sleep(5)
```

Comment:
Consider adding a `wait_for_workflow_completion` method, like we have [here in the .NET SDK](https://github.com/dapr/dotnet-sdk/blob/master/src/Dapr.Client/DaprClient.cs#L1051). This is also similar to the `d.wait(5)` method that you're using above to wait for the dapr sidecar to start.